agent/app/agent_capability.go

```diff
@@ -63,7 +66,12 @@ func (agent *ecsAgent) capabilities() []*ecs.Attribute {
 	// Determine API versions to report as supported. Supported versions are also used for capability-enablement, except
 	// logging drivers.
 	for _, version := range agent.dockerClient.SupportedVersions() {
-		capabilities = appendNameOnlyAttribute(capabilities, capabilityPrefix+"docker-remote-api."+string(version))
+		dockerVersion, _ := strconv.ParseFloat(string(version), 64)
```

Comment:
We should probably not parse this as a float.  1.2 is not greater than 1.11.

Reply:
aah yes, will change it.